app/views/bookingsync_portal/admin/rentals/index.html.erb

```diff
@@ -44,12 +46,12 @@
             </div>
           <%- end -%>
           <%- if BookingsyncPortal.create_remote_rental -%>
-            <%= render "new_remote_rental", remote_account: remote_account %>
+            <%= render "new_channel_listing", channel_listing_section: channel_listing_section %>
           <%- end -%>
         <% end %>
       </div>
       <div class="rentals-list-footer text-center">
-        <%= render partial: 'new_remote_account' %>
+        <%= render partial: 'new_channel_listing_section' %>
```

Comment:
@kaizencodes perhaps this code may be used instead `property_creation_request` in expedia. WDYT?

Reply:
@sineed yeah probably, you developed the ui for it so you have more insight on this.